lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/LibertyWorkspace.java

```diff
@@ -118,6 +120,15 @@ public void scanXMLforInclude(Path filePath) {
         }
     }
 
+    public boolean isConfigDir(Path filePath, BasicFileAttributes fileAttributes) {
+        if (!fileAttributes.isDirectory()) {
+            return false;
+        }
+        return filePath.endsWith(LibertyConstants.WLP_USER_CONFIG_DIR) || 
```

Comment:
Can filePath be a file that is within one of these dirs, and if so how do they get added to the config file list?